docs/content/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md

```diff
@@ -55,6 +55,26 @@ put it under `<FLINK_HOME>/lib/`.
 [flink-sql-connector-db2-cdc](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-db2-cdc),
 more released versions will be available in the Maven central warehouse.
 
+Since Db2 Connector's IPLA license is incompatible with Flink CDC project, we can't provide Db2 connector in prebuilt connector jar packages. 
+You may need to configure the following dependencies manually.
```

Comment:
What about providing a link for downloading the JAR we cannot provide? Same for all connectors with license issues.